runtime/lua/vim/health.lua

```diff
@@ -307,32 +308,37 @@ local function shellify(cmd)
   return table.concat(escaped, ' ')
 end
 
+--- @private
 function M.cmd_ok(cmd)
   local out = vim.fn.system(cmd)
   return vim.v.shell_error == 0, out
 end
 
+--- @private
 --- Run a system command and timeout after 30 seconds.
 ---
---- @param cmd table List of command arguments to execute
---- @param args ?table Optional arguments:
+--- @param cmd string[] Command arguments to execute
+--- @param args? table Optional arguments:
```

Comment:
I think mixing the variable name and its typing like this looks bad. I know I wrote this, but let's switch this.

```suggestion
--- @param args table? Optional arguments:
```

Reply:
Oh wait, is the difference that the `?` in parameter name indicates that it's an optional argument? OK, maybe this makes more sense in that case. Ignore this comment.

Reply:
Yes, that is it precisely.